pkg/cmd/workflow.go

```diff
@@ -248,7 +248,7 @@ func newRunWorkflowCommand() *cobra.Command {
 		RunE:  doRunWorkflow,
 	}
 
-	cmd.Flags().StringArray("parameter", []string{}, "Parameters to invoke this workflow run with")
+	cmd.Flags().StringArrayP("parameter", "p", []string{}, "Parameters to invoke this workflow run with")
```

Comment:
This is good, we should standardize on adding shortcuts. Let's do a quick check to make sure we're doing this everywhere?

Reply:
I did a quick check and everything looks good--we're now using short hand for flags everywhere with this change.

Reply:
👍